pages/linux/mashtree.md

```diff
@@ -0,0 +1,17 @@
+# mashtree
+
+> Makes a fast tree from genomes.
+> Does not make a phylogeny.
+> More information: <https://github.com/lskatz/mashtree>.
+
+- Fastest method:
```

Comment:
We should explain what the example does. maybe:
```suggestion
- Create a tree from a X file using multiple thread and store into a DND file:
```

Reply:
Thank you.  I added better descriptions.